### ISSUE TITLE
Native tokenFrom causes an error in executeBridgeTokenTransaction

### ISSUE THREAD

**Issue by Vaulverin** [synapsecns/sdk#41]
https://github.com/synapsecns/sdk/blob/aa2d65535dd15e9ebbd6756251bb146a3b293cfc/src/bridge/bridge.ts#L245
There is a function checkCanBridge that can be applied for ERC20 tokens only. If I pass native token, the error occurs.
I've tried to skip those checks and bridge completed successfully.

**Comment by BlazeWasHere** [comment]
Can you send a minimal reproducible code?

**Comment by Vaulverin** [comment]
```typescript
import {ethers, providers, Wallet} from 'ethers';
import {Bridge, Networks, Tokens} from '@synapseprotocol/sdk';

const WALLET_PRIVATE_KEY = '';
async function main() {
  const provider = new providers.JsonRpcProvider(process.env.ARBITRUM_RPC);
  const walletFrom = new Wallet(WALLET_PRIVATE_KEY, provider);
  const fromNetwork = Networks.ARBITRUM;
  const toNetwork = Networks.AVALANCHE;
  const tokenFrom = Tokens.ETH;
  const tokenTo = Tokens.WETH_E;
  const amountFrom = ethers.constants.WeiPerEther;
  const synapseBridge = new Bridge.SynapseBridge({
    network: fromNetwork.chainId,
    provider: walletFrom.provider,
  });
  const amountTo = await synapseBridge.estimateBridgeTokenOutput({
    tokenFrom,
    chainIdTo: toNetwork.chainId,
    tokenTo,
    amountFrom,
  });
  /** the error occurs in executeBridgeTokenTransaction
   *  reason: 'invalid contract address or ENS name',
   *   code: 'INVALID_ARGUMENT',
   *   argument: 'addressOrName',
   *   value: null
   */
  const tx = await synapseBridge.executeBridgeTokenTransaction(
    {
      amountFrom,
      amountTo: amountTo.amountToReceive,
      chainIdTo: fromNetwork.chainId,
      tokenFrom,
      tokenTo,
    },
    walletFrom
  );
  await tx.wait();
  console.log(`Success: ${tx.hash}`);
}

main();

```

**Comment by bleaker0x** [comment]
@Vaulverin extreme apologies for not getting back to you far sooner on this, for some reason I wasn't notified that your issues had been opened up. 

A function for this exact thing is in our dev branch right now (https://github.com/synapsecns/sdk/blob/dev/src/tokenswap/tokenswap.ts#L76), and following a merge from `master` this weekend you'll be able to use that function.

**Comment by bleaker0x** [comment]
@Vaulverin apologies for not seeing this issue (along with your other one) sooner. So, from what I can see, you're passing `chainIdTo: fromNetwork.chainId` to `executeBridgeTokenTransaction` which would lead to the `"invalid contract address or ENS name"` error you're receiving, since WETH.e doesn't exist on Arbitrum. 

Let me know if you still have an issue after passing `toNetwork.chainId` to the `chainIdTo` parameter.

**Comment by Vaulverin** [comment]
Yeah, it's just a misspell in this example. With `toNetwork.chainId` i have the same error.

**Comment by BlazeWasHere** [comment]
It seems like you made a fork and fixed it with this [commit](https://github.com/Vaulverin/synapse-sdk/commit/be2f8a84e52e4856f5193b91ff4c34350abef521), could you make a PR into the `dev` branch with that?

**Comment by Vaulverin** [comment]
Yeah, sure

**Comment by bleaker0x** [comment]
Closing as fixed with merge of #58 into `dev`.
